### ISSUE TITLE
Question about the input size of the model

### ISSUE THREAD

**Issue by FandosA** [AnTao97/dgcnn.pytorch#59]
Hello, first of all thank you very much @AnTao97 for this repository. I just have a couple of brief, simple questions about the input size of the segmentation model that, after reading carefully the paper, I don't understand.

- The input size of the first ``nn.Conv2d`` layer of the part segmentation model is 6, is this because the input parameters to the model are one point (x, y, z) and its respective normals (nx, ny, nz)? If it's like this, then the input vector to the neural network is (x, y, z, nx, ny, nz) right?
- And second, respect to the semantic segmentation model, why the input size in this case is 18? In the paper they specify that the input vector is 9D (XYZ, RGB and normalized spatial coordinates)

Thank you very much and sorry for the inconvenience.

**Comment by antao97** [comment]
In part segmentation, the point feature is a 3-dim vector (x, y, z). In function `get_graph_feature` the 3-dim vector is extended into a 6\times k matrix by graph feature extraction. The k represents the k neighbors around this point. 
https://github.com/AnTao97/dgcnn.pytorch/blob/97785863ff7a82da8e2abe0945cf8d12c9cc6c18/model.py#L272 

Please refer to more details of graph feature extraction in code: https://github.com/AnTao97/dgcnn.pytorch/blob/97785863ff7a82da8e2abe0945cf8d12c9cc6c18/model.py#L36-L62

**Comment by FandosA** [comment]
Oh god, that's true. I didn't realise about it. thank you very much!
